### ISSUE TITLE
Custom colors not used in Material-Theme

### ISSUE THREAD

**Issue by Springrbua** [vaadin/vaadin-login#89]
### Problem
If I change the `--material-primary-color` CSS-Variable, the `vaadin-login-form` still displays the original color.
This is probably because the [vaadin-login-form-wrapper-styles](https://github.com/vaadin/vaadin-login/blob/master/theme/material/vaadin-login-form-wrapper-styles.html) include the `material-color-light` dom-module, which is therefore added to the shadow-dom of the `login-form-wrapper` and overwrites the custom value.

### Example
https://stackblitz.com/edit/vaadin-login-material-issue?file=index.html

Basically I change the value of `--material-primary-color` to yellow for the `body` but the login-button still uses the original deep-purple.

### Notes
The same example works for the lumo theme.
This is probably because the [color.html for lumo](https://github.com/vaadin/vaadin-lumo-styles/blob/master/color.html) only overwrites the dark-theme colors in the dom-module, while the [color.html for material](https://github.com/vaadin/vaadin-material-styles/blob/master/color.html) define all colors (dark and light) in the dom-module

**Comment by DiegoCardoso** [comment]
Thanks for reporting this issue. I didn't have enough time to give a deep research on this, but I suspect the problem is indeed that we are including the material colors module into our styles. 

I hope we can return to this bug soon.

**Comment by Springrbua** [comment]
@DiegoCardoso I just had a look at [`vaadin-button-styles`](https://github.com/vaadin/vaadin-button/blob/master/theme/material/vaadin-button-styles.html) which, obviously, is also using the colors and typography. However, it does not include those styles and just expects them "to be there".
So it is probably really enough to just remove this `include`.

**Comment by DiegoCardoso** [comment]
Thanks again for your report. This change is part of `v1.0.1`.
